.github/workflows/test.yml

```diff
@@ -14,12 +16,17 @@ jobs:
     strategy:
       matrix:
         otp: [21.x, 22.x]
-        elixir: [1.7.x, 1.8.x, 1.9.x]
+        elixir: [1.7.x, 1.8.x, 1.9.x, 1.10.x]
 
     steps:
       - uses: actions/checkout@v1.0.0
 
-      - uses: actions/setup-elixir@v1.1.0
+      - name: set environment variables
+        uses: allenevans/set-env@v1.0.0
+        with:
+          CACHE_VERSION: 'rzMJIkRUIxebxwr4'
```

Comment:
It would be great to remove this magic string, but I guess there is no better way to reset cache on actions.